common_functions.sh

```diff
@@ -336,6 +350,12 @@ function get_sums_for_build_arch() {
 		# If there are multiple builds for a single version, then pick the latest one.
 		shasums_url=$(cat ${shasum_file} | grep "checksum_link" | head -1 | awk -F'"' '{ print $4 }');
 		shasum=$(curl -Ls ${shasums_url} | sed -e 's/<[^>]*>//g' | awk '{ print $1 }');
+		arch_build_version=$(cat ${shasum_file} | grep "release_name" | awk -F'"' '{ print $4 }');
+		arch_build_version=$(get_nightly_short_version ${gsba_build} ${arch_build_version})
+		# Only print the arch related info if the arch version matches with the parent
+		if [ "${arch_build_version}" != "${full_version}" ]; then
+			continue;
```

Comment:
Block doesn't execute what the comment says?

Reply:
Maybe I have to update the comment, if the versions dont match, it skips to the end of the loop and prints only when the two match.

Reply:
Yeah comment update I think in that case :-)

Reply:
Updated the comment now.